web-api/src/personalization/routes.rs

```diff
@@ -131,11 +131,17 @@ pub(crate) async fn update_interactions(
     Ok(())
 }
 
+const fn default_true() -> bool {
```

Comment:
Usually, we call them with the name of the property, not the value. `default_include_properties`.